test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -103,4 +103,10 @@ Describe "TabCompletion" -Tags CI {
         $res = TabExpansion2 -inputScript 'using nam' -cursorColumn 'using nam'.Length
         $res.CompletionMatches[0].CompletionText | Should be 'namespace'
     }
+
+    It 'Should complete "Export-Counter -FileFormat" with available output formats' {
+        $res = TabExpansion2 -inputScript 'Export-Counter -FileFormat ' -cursorColumn 'Export-Counter -FileFormat '.Length
+        $res.CompletionMatches.Count | Should Be 3
+        $res.CompletionMatches.Foreach{$_.CompletionText -in 'blg', 'csv', 'tsv' | Should Be $true}
```

Comment:
We try to avoid `Should Be $true` because it is difficult to understand what went wrong just reading the logs.

In this specific case, I would instead use:

```powershell
$rest.CompletionMatches.CompletionText -join ' ' | Should Be 'blg csv tsv'
```